### DISCUSSION TITLE
security policy / reporting security issues

### DISCUSSION THREAD

**Discussion by 4390c336** [xerial/sqlite-jdbc#892] [Dev]
I want to share some security findings, but the repo has no security contact or a safe way to communicate.

is there safe channel to report ? or should the project enable something like https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability

**Comment by gotson** [comment]
i cannot manage the repo's settings, @xerial would you want to enable private vulnerability reporting?

**Reply by 4390c336** [reply]
can I email the details to you or any other maintainer ?

**Reply by xerial** [reply]
Thanks. I'll take a look

**Reply by xerial** [reply]
And also enabled private reporting https://github.com/xerial/sqlite-jdbc/security/
